cmake/projects/PROJ4/hunter.cmake

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    PROJ4
+    VERSION
+    5.0.0
```

Comment:
Should it be 5.0.1?

Reply:
It is kinda confusing but 5.0.0 seems to be right. Have a look [here](https://github.com/OSGeo/proj.4/issues/976)